project.clj

```diff
@@ -1,6 +1,8 @@
 (def version "3.0.6")
+(def uberjar-name (str "firebolt.metabase-driver-" version ".jar"))
+(def uberjar-file (str "target/uberjar/" uberjar-name))
 
-(defproject io.firebolt/metabase-firebolt-driver version
+(defproject io.firebolt/firebolt.metabase-driver version
```

Comment:
Are we fine with renaming the project here? Will it cause any incompatibility issues?

Reply:
That's actually renaming it back. It used to be called so, I though this one would be better but it seems to be naming convention for metabase drivers